modules/server_new/src/main/scala/observe/server/gws/package.scala

```diff
@@ -0,0 +1,22 @@
+package observe.server
+
+import coulomb.*
+import coulomb.define.*
+
+package object gws {
+
+  export coulomb.units.accepted.{Millibar, ctx_unit_Millibar}
+
+  final type Bar
```

Comment:
We have some of these in core. I suggest we move these there

Reply:
Where else would those be used?
I guess that some reporting tools will show weather conditions